tests/test_dir_traversal.py

```diff
@@ -261,7 +273,22 @@ class DirectoryTraversalVector(Generic[T]):
 
 
 @pytest.mark.parametrize(
-    "vector", [pytest.param(v, id=v.id) for v in directory_traversal_vectors]
+    "vector",
+    [
+        pytest.param(
+            v,
+            id=v.id,
+            marks=pytest.mark.skipif(
+                platform.system() == "Windows"
+                and any(
+                    isinstance(entry, (RelativeSymlink, AbsoluteSymlink))
+                    for entry in v.given
+                ),
+                reason="Symlinks on Windows may be created only by administrators",
+            ),
```

Comment:
Nice! This condition is quite complex, but I'm happy with how readable this still is.